### ISSUE TITLE
sensu 1.6.0 doesn't execute checks - 1.5.0 is fine

### ISSUE THREAD

**Issue by peterhoeg** [sensu/sensu#1917]
We use sensu quite extensively for monitoring customer infrastructure and at any given point have a mix of versions live with only the sensu server to be guaranteed at the latest version.

When 1.6.0 was released we deployed that on a few sites for testing but realized that the checks simply stopped running. Both reverting to the old environment with 1.5.0 as well as changing the new environment to use 1.5.0 made the checks run again:

 - iteration 1, sensu 1.5.0 on ruby 2.4 with config version 1 (working)
 - iteration 2, sensu 1.6.0 on ruby 2.5 with config version 2 (not working)
 - iteration 3, sensu 1.5.0 on ruby 2.5 with config version 2 (working)
 
We went back to 1.5.0 to not be blind, so I will try to do a new deployment on a proper test site to capture some logs.

## Expected Behavior

1.6.0 to run checks normally.

## Current Behavior

Checks simply don't run - not even keepalive despite the sensu-client process running.

## Your Environment

* Sensu version used: 1.5.0 and 1.6.0
* Active Sensu extensions influxdb2 (server), snmp-trap and system-profile (client)
* Operating System and version NixOS 18.09
* Transport and runtime versions RabbitMQ 3.6.15 on Erlang 20.3.8.9

**Comment by nadx969** [comment]
We are seeing the exact same issue.  Upgraded sensu-server from 1.4.3 to 1.6.0 yesterday 
OS: AMI Linux 1 with embedded ruby 2.4
Transport: Redis (AWS Elasticache) 

Only difference between our issue and @peterhoeg  is the Keepalives and sensu-extensions-system-profile checks are working, however none of the others.  when running a client with logging set to debug, can see the client subscribe to the correct subscriptions only the keepalives get executed.

Noted that when doing `yum info sensu` the version is 1.5.0 however in the Uchiwa dashboard, the client version indicates 1.6.0

**Comment by portertech** [comment]
The release diff https://github.com/sensu/sensu/compare/v1.5.0...v1.6.0

**Comment by portertech** [comment]
I am not able to reproduce the issue on my local (using the Rabbit transport), the client executes checks as expected. The release/changes are quite minimal, with the exception of the Rabbit transport connection/channel changes. I am not sure why @nadx969 would have troubles when using the Redis transport.

**Comment by nadx969** [comment]
this was a bizarre situation.  
we did an upgrade a couple of days ago.  and as described, all the checks stopped executing on all the clients.   after deleting the clients via the api, upgrading gems, uninstalling and reinstalling various versions of sensu, downgrading sensu, deploying new sensu fresh instance, I determined the common denominator was the Redis database.  after a quick test of changing the the redis db and seeing normal operations, I ran this after connecting to our redis cluster:
`redis-cli> FLUSHALL`
everything returned to normal.  unfortunately, there was nothing in the logs to support that there were any issues with Redis, all clients registered with their subscriptions 
and the Redis cluster is AWS Elasticache so some issues can go easily unseen and i was more interested in getting Sensu back up 

suffice to say, we are back and our issue is not related to this one.
